protos/start_pos.proto

```diff
@@ -0,0 +1,14 @@
+//Message for communicating starting position
+
+syntax = "proto3";
+
+option optimize_for = LITE_RUNTIME;
+
+enum Pos {       //we can add more start positions in the future
+	LEFT = 0;
+	RIGHT = 1;
+}
+
+message StartPos {
+	Pos pos = 1;
+}
```

Comment:
Yes I believe it's from the perspective of one team. One thing though is that I don't think we have that information stored anywhere on Dawn yet, so I just left the `sendRobotStartPos` function kind of as a placeholder until we develop that.

Reply:
Yeah, I'm not really sure exactly how this is going to function, but the idea is that this information can be made available to the student code so students don't have to copy-paste their autonomous code depending on their starting location on the field. Idea first proposed by Shepherd, and Runtime decided it was a good idea so it's just implemented here, the names can be better than just `LEFT` and `RIGHT` in the future :P

Reply:
Ah yes, so about this. I was thinking, what if in the Dawn interface we made different windows for the coding challenges, autonomous code, and teleop code.

Reply:
Right now, we currently have everything in one file, but in terms of design I'm not sure if we would have separate files for each one or if we could have 1 file and section off each part somehow.

Reply:
🤔 🤔 
I really like that idea, if you think it wouldn't require a major overhaul to pull off. If you changed it to upload multiple files to Runtime, I think Runtime would be able to handle that pretty easily as well (and it actually might even be cleaner than the code currently is)

Reply:
It might be doable. Separating editor.js(or any react files) into multiple files, I think, won't be too much of a problem. Having multiple windows is more on the electron side than react, but I think it is possible to manage to do it.

Reply:
Yea, I think this might definitely be a big feature though, so I think getting the refactoring in order first might be better. In the meantime, we can keep brainstorming how to design this well.

Reply:
I think it would be better if the auton and teleop were in file, and challenges were in another. This would be much easier to implement in Runtime, and probably makes more sense.